app/views/tag/index.html.erb

```diff
@@ -80,3 +83,21 @@
 </div>
 
 <div class="text-center"> <%= will_paginate @tags, :renderer => BootstrapPagination::Rails if @paginated %></div>
+
+<script type="text/javascript">
```

Comment:
Here, I'd like to propose that we make this into a separate JavaScript file, and a function which accepts a jQuery selector like:

```js
setupRemoteFollow($('follow-btn-remote'));
```

What it can do is read the `href` attribute of each `<a class="follow-btn-remote">` tag on the page, and parse it to set up the remote call. 

Then we include this JS file on all pages. Then all we have to do to implement the remote call is to give any link the `follow-btn-remote` class, and it gets hooked up automatically. And if JavaScript fails, it will revert to being a normal link that doesn't require JS.

How does that sound?

Reply:
Yes sure sounds good, I will do that !

Reply:
@jywarren I am trying to implement this. We are trying to generalize and use in this file in various remote requests right? But each one would have different requirements, like in this case we need to change the 'follow' button to 'following', etc. In this case how can we implement this? Thank you !